src/utils/release-utils.ts

```diff
@@ -19,3 +19,26 @@ export const getTenantPipelineRunFromRelease = (release: ReleaseKind): string =>
 export const getFinalPipelineRunFromRelease = (release: ReleaseKind): string => {
   return release.status?.finalProcessing?.pipelineRun;
 };
+
+// Example:
+// Math.random().toString(36)               // "0.f5b27xpmc"
+// Math.random().toString(36).slice(2, 7)   // "f5b27"
+const generateSuffix = (length = 5): string =>
+  Math.random()
+    .toString(36)
+    .slice(2, 2 + length);
```

Comment:
Do we need this specifically? We can set `metadata.generateName = prefix` while creating the resource, that should do it for you in most cases

Reply:
hmm, good question - I'll give it a try to update the `metadata.generateName` instead of `name` (without this "suffix" logic) and check if it works :)